swan-lake/learn-the-platform/ballerina-persist/persist-client-api.md

```diff
@@ -0,0 +1,245 @@
+---
+layout: ballerina-persist-documentation-left-nav-pages-swanlake
+title: Persist Client API
+description: The sections gives details on the generated Persist Client API.
+keywords: ballerina, programming language, ballerina packages, persist, persist client, persist client api
+permalink: /learn/ballerina-persist/persist-client-api/
+active: persist_client_api
+intro: Persist client is used to perform CRUD operations on the data source. Each entity type will have five resource methods for each operation. The resource methods are get, get(get by identity), post, put, and delete. Persist CLI tool will generate the derived Entity Types and the Clients from the model definition.
```

Comment:
```suggestion
intro: The Persist Client is used to perform CRUD operations on the data source. Each entity type will have five resource methods for each operation. The resource methods are `get`, `get(get by identity)`, `post`, `put`, and `delete`. The Persist CLI tool will generate the derived entity types and the clients from the model definition.
```
Shall we remove the sentence with the words within backticks out of the intro as backticks will not be rendered properly when they are added in the front matter.